subprojects/inject-generics/inject-generics.gradle

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+apply from: "$rootDir/gradle/dependencies.gradle"
+apply from: "$rootDir/gradle/java-library.gradle"
+apply from: "$rootDir/gradle/root/coverage.gradle"
+
+description = "Filter @InjectMocks according to Java's assignment rules for generics"
```

Comment:
Should we make this artifact name `inject-mocks` instead? The longterm plan would be to move `@InjectMocks` into this package so that users can rely on that instead. While the current artifact would solve the problem with generics, I ultimately want to move `@InjectMocks` here and move all related machinery into one place.